oozie/src/main/java/org/apache/falcon/oozie/DatabaseExportWorkflowBuilder.java

```diff
@@ -169,4 +182,50 @@ private StringBuilder buildNumMappers(StringBuilder builder, Map<String, String>
         org.apache.falcon.entity.v0.feed.Cluster feedCluster = FeedHelper.getCluster(entity, cluster.getName());
         return FeedHelper.getExportArguments(feedCluster);
     }
+
+    private StringBuilder buildExportTableArg(StringBuilder builder, CatalogTable catalog) throws FalconException {
+
+        LOG.info("Catalog URI {}", catalog.getUri());
+        builder.append("--skip-dist-cache").append(ImportExportCommon.ARG_SEPARATOR);
+        Iterator<String> itr = Splitter.on("#").split(catalog.getUri()).iterator();
+        String dbTable = itr.next();
+        String partitions = itr.next();
```

Comment:
Can there be no partitions? If yes itr.next() will return null and getPartitionKeyValues method will throw exception. If partitions are always provided, "if" condition in line 207 seems unnecessary